client/my-sites/customer-home/locations/secondary/index.jsx

```diff
@@ -88,6 +88,9 @@ const Secondary = ( { cards, siteId, trackFirstCardAsPrimary = false } ) => {
 							onPageSelected={ trackInnerCardImpression }
 						>
 							{ card.map( ( innerCard, innerIndex ) => {
+								if ( CARD_COMPONENTS[ innerCard ].isDisabled === true ) {
```

Comment:
I am not a big fan of this shortcut, but I think it's OK if we remove it in a follow-up once the translations are all available.

Reply:
That's a good idea. There's at least another follow-up to be made to remove the card from the first position (as per [comment](https://github.com/Automattic/wp-calypso/pull/91551#discussion_r1633005146)).
We can remove it then if we don't get to it earlier.